ODataValidation/Controllers/Customer.cs

```diff
@@ -5,4 +5,12 @@ public class Customer
     public int Id { get; set; }
     public string Name { get; set; }
     public int Age { get; set; }
-}
\ No newline at end of file
+}
+
+public class Company
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public string Address {get;set;}
+    public DateTime Established {get;set;}
```

Comment:
/wtd use DateOnly, fix formatting

Reply:
@coderabbitai use DateOnly, fix formatting

Reply:
@coderabbitai can you also add xml docs to class?

Reply:
@codrabbit can you convert class to `record` with primary constructor and keep all enhancements already done?

Reply:
@codrabbitai can you convert class to record with primary constructor and keep all enhancements already done?